src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -101,6 +102,14 @@ public SliceFromConfig(final RepoConfig config, final FileSystem fs) {
                             );
                         }
                     ),
+                    new MapEntry<>(
+                        "nuget",
+                        config -> cfg.url().thenCompose(
```

Comment:
@olegmoz  let's extract `NuGet` creation to a private method to make code more readable